config/core/test-configs.yaml

```diff
@@ -2099,6 +2102,7 @@ test_configs:
     test_plans:
       - baseline
       - baseline-nfs
+      - kselftest-seccomp
```

Comment:
The `jetson-tk1` is very old and there's only one left online, so I would rather not add anything to it. So it's probably best to drop this.